### DISCUSSION TITLE
How to display the "last updated" and "created" dates at the bottom of an article?

### DISCUSSION THREAD

**Discussion by jygzyc** [squidfunk/mkdocs-material#6968] [Q&A] [answered]
As the title suggests, how can I create an effect similar to the image shown below?

![Screenshot_20240325_153001.jpg](https://github.com/squidfunk/mkdocs-material/assets/33087260/284ea9c0-a822-41cb-9173-11bb566b4f68)

**Comment by alexvoss** [comment, marked as answer]
This is [metadata pulled from a Git repository](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/). Specifically, the [document dates](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-dates) part explains how to get the creation and update dates.
